src/Features/Blockcore.Features.WebHost/Startup.cs

```diff
@@ -91,39 +111,67 @@ public void ConfigureServices(IServiceCollection services)
                 .AddNewtonsoftJson(options => Serializer.RegisterFrontConverters(options.SerializerSettings))
                 .AddControllers(this.fullNode.Services.Features, services);
 
-            // Enable API versioning.
-            // Note much of this is borrowed from https://github.com/microsoft/aspnet-api-versioning/blob/master/samples/aspnetcore/SwaggerSample/Startup.cs
-            services.AddApiVersioning(options =>
-            {
-                // Our versions are configured to be set via URL path, no need to read from querystring etc.
-                options.ApiVersionReader = new UrlSegmentApiVersionReader();
-
-                // When no API version is specified, redirect to version 1.
-                options.AssumeDefaultVersionWhenUnspecified = true;
-            });
-
-            // Add the versioned API explorer, which adds the IApiVersionDescriptionProvider service and allows Swagger integration.
-            services.AddVersionedApiExplorer(
+                services.AddSwaggerGen(
                 options =>
                 {
-                    // Format the version as "'v'major[.minor][-status]"
-                    options.GroupNameFormat = "'v'VVV";
+                    string assemblyVersion = typeof(Startup).Assembly.GetName().Version.ToString();
+
+                    options.SwaggerDoc("node",
+                           new OpenApiInfo
+                           {
+                               Title = "Blockcore Node API",
+                               Version = assemblyVersion,
+                               Description = "Access to the Blockcore Node features.",
+                               Contact = new OpenApiContact
+                               {
+                                   Name = "Blockcore",
+                                   Url = new Uri("https://www.blockcore.net/")
+                               }
+                           });
+
+                    string[] ApiXmlDocuments = new string[]
```

Comment:
I see the problem if this is really important it could be loaded from an interface perhaps (or a DocumentsFactory)?

Reply:
We'll find another solution to it, this PR will at least add documentation to a lot more APIs, but not all of them.

Reply:
it could scaffold the folder